### ISSUE TITLE
swagger-ui route in SPA

### ISSUE THREAD

**Issue by agrakhov** [swagger-api/swagger-ui#5525]
### Q&A (please complete the following information)
 - OS: [macOS]
 - Browser: [chrome]
 - Version: [latest]
 - Method of installation: [npm]
 - Swagger-UI version: [3.23.4]
 - Swagger/OpenAPI version: [OpenAPI 3.0]

### Content & configuration

Swagger/OpenAPI definition: petstore oas 3 (not important)
Swagger-UI configuration options (very basic, nothing specific):
```js
SwaggerUI({
  dom_id: '#swaggerContainer',
  spec: swagger,
  docExpansion: 'list',
});
```
### Screenshots
none

### How can we help?
We are rendering SwaggerUI one of our pages in SPA (React + React-router), lets say `https://my.awesome.app/path/to/page/1` but as soon as we navigates to this page swagger component changing route to `https://my.awesome.app/#` which is breaking our users experience. 
Any guidelines / hints how to prevent this from happening?

**Comment by shockey** [comment]
Hi @agrakhov! Are you sure you aren't setting the `deepLinking: true` option?

**Comment by agrakhov** [comment]
@shockey yup I've tried to explicitly set this to `deepLinking: false` but it still changing the url.

**Comment by shockey** [comment]
Strange. Some questions:

* You're sure this is coming from Swagger UI? You've tried commenting out the Swagger UI call and the `#` is no longer pushed to your history?
* Have you tried using `swagger-ui-react`? I don't _think_ it'd cause a difference, but perhaps React Router isn't liking a different app being mounted within a component tree it manages.
